targets/unity-v2/Testing/Tests/Entity/EntityApiTests.cs

```diff
@@ -116,23 +126,23 @@ private void GetObjectCallback1(GetObjectsResponse result)
 
             _testInteger = (_testInteger + 1) % 100; // This test is about the Expected value changing - but not testing more complicated issues like bounds
 
-            var updateRequest = new SetObjectsRequest
+            var updateRequest = new DataModels.SetObjectsRequest
             {
-                Entity = _entityKey,
-                Objects = new List<SetObject> {
-                    new SetObject{ ObjectName = TEST_OBJ_NAME, DataObject = _testInteger }
+                Entity = new DataModels.EntityKey { Id = _entityId, TypeString = _entityTypeString },
+                Objects = new List<DataModels.SetObject> {
+                    new DataModels.SetObject{ ObjectName = TEST_OBJ_NAME, DataObject = _testInteger }
                 }
             };
-            PlayFabEntityAPI.SetObjects(updateRequest, PlayFabUUnitUtils.ApiActionWrapper<SetObjectsResponse>(testContext, UpdateObjectCallback), PlayFabUUnitUtils.ApiActionWrapper<PlayFabError>(testContext, SharedErrorCallback), testContext);
+            PlayFabDataAPI.SetObjects(updateRequest, PlayFabUUnitUtils.ApiActionWrapper<DataModels.SetObjectsResponse>(testContext, UpdateObjectCallback), PlayFabUUnitUtils.ApiActionWrapper<PlayFabError>(testContext, SharedErrorCallback), testContext);
         }
-        private void UpdateObjectCallback(SetObjectsResponse result)
+        private void UpdateObjectCallback(DataModels.SetObjectsResponse result)
```

Comment:
a bit of a nitpick here: may be worth having using namespace DataModels? Makes the function declaration a bit cleaner.

Reply:
I'm at a bit of a crossroads on that thinking.
Certainly a customer can do it if/when they want to.
For us:
There are name overlaps in MANY of the model namespaces at this point, and that's only going to get worse.
Those places are mandatory for defining the model namespace, and so, I'm leaning towards consistency, where internally we define all the namespaces.

Also...

There's a task where we remove all using's in our own plugin to avoid namespace collision problems with customers who define classes of the same names in the global space by accident, and then see compiler errors in our code, despite it not being our problem. So, this might just get worse for us all around.